.github/configs/alloy-config.yaml

```diff
@@ -1,31 +1,31 @@
-# This config is for a stand-alone Grafana Agent that will generate metrics, logs, and traces and send them to the
-# k8s monitoring-deployed Grafana Agent via its receivers.
-agent:
+# This config is for a stand-alone Grafana Alloy that will generate metrics, logs, and traces and send them to the
+# k8s monitoring-deployed Alloy via its receivers.
+alloy:
   configMap:
     content: |-
       prometheus.scrape "myself" {
-        job_name = "integrations/agent"
+        job_name = "integrations/alloy"
         targets = [{"__address__" = "localhost:80"}]
```

Comment:
FWIW the default listen address in the new Helm chart is 0.0.0.0:12345 (to align with the default port everywhere else); you'll want to make sure you set it back to port 80 if you want this to continue working.

Reply:
Thanks for the note. Updated the port to 12345